src/templates/base/2019/base.html

```diff
@@ -278,7 +305,7 @@
             </li>
           </ul>
         </nav>
-        <nav aria-label="{{ self.menu_title() }}" id="menu" aria-labelledby="menu-btn">
+        <nav aria-label="{{ self.menu_title() }}" id="menu" aria-labelledby="menu-btn" class="js-show">
```

Comment:
@bazzadp 
I prefer to go to the footer navigation item rather than hide the hamburger menu if javascript is disabled.

Options:
However, it can be annoying to return to the top after moving to the footer.
For mobile, do you think it would be nice to have a link in the footer that takes you to the top of the page as well?

Reply:
Good idea - done.

I'll leave the scroll to top link for now as a bit out of scope for this and as Rick says the non-JS version should be used by very few users! I know iOS has easy scroll to top by tapping on menu in browser. Not sure if Android does but from a quick Google it appears not!